pkgs/development/tools/protoscope/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "protoscope";
+  version = "unstable-2022-06-30";
+  rev = "aed2b19b38635bae50b4c1583223c2b316fec329";
+
+  src = fetchFromGitHub {
+    owner = "protocolbuffers";
+    repo = "protoscope";
+    inherit rev;
+    sha256 = "sha256-nGVM1v7bqv/14wcyEtBs0EZ6jcz4ODwEGn/dBSt0nh8=";
+  };
```

Comment:
`rev` can be put inside `fetchFromGitHub` instead of `mkDerivation` as the build process doesn't requires it.

Reply:
Done